build.gradle.kts

```diff
@@ -60,11 +64,8 @@ dependencyCheck {
 
 publishing {
     publications {
-        register<MavenPublication>(rootProject.name) {
-            groupId = project.group as? String
-            artifactId = project.name
-            version = project.version as? String
-            from(components["java"])
+        withType<MavenPublication> {
```

Comment:
Did the publication name change? if so, the CI or release might need to be updated.

Reply:
It doesn't changed and `publish-snapshot.yml` should work. I tested on github release in our fork (changed only target repo)

Reply:
It looks like we are missing the publishing names - I am attempting to revert this change to see if I can still publish